pandas/core/aggregation.py

```diff
@@ -196,3 +266,49 @@ def maybe_mangle_lambdas(agg_spec: Any) -> Any:
         mangled_aggspec = _managle_lambda_list(agg_spec)
 
     return mangled_aggspec
+
+
+def _relabel_result(
+    result: Union[ABCSeries, ABCDataFrame],
```

Comment:
You can use FrameOrSeriesUnion from pandas._typing, or maybe even FrameOrSeries if applicable here

Reply:
can you update for this

Reply:
done